.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: go
+
+services:
+  - mongodb
+
+go:
+  - 1.7.1
```

Comment:
Do we want to support older versions like 1.6 etc? What's our minimum supported version?